Dockerfile

```diff
@@ -12,36 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM dddecaf/wsgi-base:master
-
-ENV APP_USER=giraffe
-
-ARG UID=1000
-ARG GID=1000
+FROM gcr.io/dd-decaf-cfbf6/modeling-base:master
 
 ARG CWD="/app"
-
 ENV PYTHONPATH="${CWD}/src"
-
-RUN addgroup -S -g "${GID}" "${APP_USER}" && \
-    adduser -S -H -u "${UID}" -G "${APP_USER}" "${APP_USER}"
-
 WORKDIR "${CWD}"
 
-COPY requirements.in dev-requirements.in ./
+# pin pip to 18.0 to avoid issue with cobra -> depinfo -> pipdeptree -> pip._internal.get_installed_distributions
```

Comment:
This is no longer relevant. pipdeptree made another hotfix but hopefully there will also be a long-term solution.